src/workloads/readme.md

```diff
@@ -0,0 +1,2 @@
+# Building
+The workloads project can only be built using .NET Framework msbuild. To build locally, run ```build -project src\workloads\workloads.csproj -msbuildEngine vs```
```

Comment:
You could also mention that alternatively you can use the Developer Command Prompt shell on Windows.